### PR TITLE
UHF-9620: Add assets to Varnish cache, new asset purge configuration

### PR DESCRIPTION
**Pull request by tuutti**

### CODE REVIEW

File: public/sites/default/settings.php [hunk 1]
```diff
@@ -182,34 +168,66 @@ function drupal_get_env(string|array $variables) : mixed {
...
+    $config['varnish_purger.settings.' . $name]['hostname'] = $varnish_host;
+
+    if ($varnish_port = getenv('DRUPAL_VARNISH_PORT')) {
+      $config['varnish_purger.settings.' . $name]['port'] = $varnish_port;
+    }
+  }
+
+  foreach ($headers as $header) {
+    $config['varnish_purger.settings.' . $name]['headers'][] = $header;
+  }
```
Comment (hyrsky) [inline public/sites/default/settings.php:223]:
```diff
diff --git a/public/sites/default/settings.php b/public/sites/default/settings.php
index 6683510..fadca81 100755
--- a/public/sites/default/settings.php
+++ b/public/sites/default/settings.php
@@ -216,10 +216,10 @@ function drupal_get_env(string|array $variables) : mixed {
     if ($varnish_port = getenv('DRUPAL_VARNISH_PORT')) {
       $config['varnish_purger.settings.' . $name]['port'] = $varnish_port;
     }
-  }
 
-  foreach ($headers as $header) {
-    $config['varnish_purger.settings.' . $name]['headers'][] = $header;
+    foreach ($headers as $header) {
+      $config['varnish_purger.settings.' . $name]['headers'][] = $header;
+    }
   }
 
   if ($varnish_purge_key = getenv('VARNISH_PURGE_KEY')) {
```

**Review by hyrsky — CHANGES_REQUESTED** [review]
Shouldn't this be:

File: public/sites/default/settings.php (hunk 1 above)
Comment (tuutti) [reply public/sites/default/settings.php:223]: Yes it should. Thanks!

Comment (hyrsky) [reply public/sites/default/settings.php:223]: Setting the purge key should be in the loop as well.

Comment (tuutti) [reply public/sites/default/settings.php:223]: I don't think so. It has to be set to every `varnish_purger.settings.*` item regardless of what's defined in `$varnishConfiguration`

Comment (hyrsky) [reply public/sites/default/settings.php:223]: Ah, which is `varnish_purge_all`. That makes it to apply to all?

Comment (tuutti) [reply public/sites/default/settings.php]: No. I'm just being dumb :see_no_evil:  Should be fixed now

**Review by hyrsky — APPROVED** [review]

**Pull request merged**